wrapper/testapp/build.gradle.kts

```diff
@@ -15,32 +15,33 @@
 */
 
 plugins {
-    id "com.diffplug.spotless"
+    id("com.diffplug.spotless")
+    application
 }
 
-apply plugin: "java"
```

Comment:
Is the Java plugin not required if the application plugin is applied?

Reply:
[Exactly](https://docs.gradle.org/current/userguide/application_plugin.html):

> Applying the Application plugin also implicitly applies the Java plugin.

Reply:
Do you want me to move that simplification out to a separate commit, mentioning the given source?

Reply:
No, that's fine, just wanted to make sure I understood how it was magically working :)